modules/imgcodecs/src/grfmt_bmp.cpp

```diff
@@ -610,6 +610,13 @@ ImageEncoder BmpEncoder::newEncoder() const
     return makePtr<BmpEncoder>();
 }
 
+#define CHECK_WRITE(action) \
+if (!action) \
+{ \
+    strm.close(); \
```

Comment:
I believe stream will be closed automatically in its destructor.

Reply:
Done. Unified the macro.